### PR TITLE
Add Trino Fest hype post

### PR DESCRIPTION
**Pull request by colebow**

### CODE REVIEW

File: _posts/2023-06-01-trino-fest-hype-speaker-lineup.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+## The Python ecosystem
+
+We've got talks from [Fugue](https://github.com/fugue-project/fugue) and
+[Ibis](https://ibis-project.org/), two different tools that integrate Python
+with SQL, and then run that SQL on underlying data sources. Both have recently
+added Trino support, and they're excited to share their use cases and introduce
+the Trino community to the new, powerful ways you can leverage it. Trino has
+always been a SQL query engine, but with Fugue and Ibis, writing Python code to
+run Trino is suddenly a reality, and analysts and data scientists may not even
```
Comment (mosabua) [inline _posts/2023-06-01-trino-fest-hype-speaker-lineup.md:43]: to run queries with Trino

File: _posts/2023-06-01-trino-fest-hype-speaker-lineup.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+need to know much SQL to get the insights they're looking for.
+
+## Data lakes
+
+Ryan Blue, the co-founder of Iceberg working at Tabular, will be exploring how
```
Comment (mosabua) [inline _posts/2023-06-01-trino-fest-hype-speaker-lineup.md:48]: founder of Tabular...

File: _posts/2023-06-01-trino-fest-hype-speaker-lineup.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+to best write CDC (change data capture) change streams into Iceberg tables. A
```
Comment (mosabua) [inline _posts/2023-06-01-trino-fest-hype-speaker-lineup.md:49]:
```suggestion
to best write CDC (change data capture) streams into Iceberg tables. A
```

File: _posts/2023-06-01-trino-fest-hype-speaker-lineup.md [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+talk from Kevin Liu at Stripe will explore how a data engineer can monitor
+queries being run on Iceberg to catch performance outliers and understand usage
+rates. A talk from Alluxio highlights caching optimizations with Trino and data
+lakes. OneHouse is giving a talk about using Trino with Hudi, exploring how to
```
Comment (mosabua) [inline _posts/2023-06-01-trino-fest-hype-speaker-lineup.md:53]: Should we link to all these companies??

**Review by mosabua — CHANGES_REQUESTED** [review]

File: _posts/2023-06-01-trino-fest-hype-speaker-lineup.md (hunk 4 above)
Comment (colebow) [reply _posts/2023-06-01-trino-fest-hype-speaker-lineup.md]: I don't think that adds a ton of value / utility. If people are curious, they can just google.

**Pull request merged**